src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/WorkloadResolver.cs

```diff
@@ -473,5 +473,19 @@ public string GetManifestVersion(string manifestId)
                 return manifest.Version;
             }
         }
+
+        public IDictionary<string, string> GetInstalledManifests()
+        {
+            var manifests = new Dictionary<string, string>();
+            foreach ((string manifestId, Stream manifestStream) in _manifestProvider.GetManifests())
+            {
+                using (manifestStream)
+                {
+                    var manifest = WorkloadManifestReader.ReadWorkloadManifest(manifestId, manifestStream);
```

Comment:
@mhutch mentioned we should not directly use WorkloadManifestReader. Is there an API in manifest resolver to do this? if not, maybe we should add this to manifest resolver. And also there should be some nuance in resolver to find manifests, please check with Mikayla on that too.

Reply:
My understanding was that the manifest reader should only be used directly in the workload resolver, but please correct me if I'm wrong.